src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/UlidToStringTransformerTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests\Extension\Core\DataTransformer;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\Extension\Core\DataTransformer\UlidToStringTransformer;
+use Symfony\Component\Uid\Ulid;
+
+class UlidToStringTransformerTest extends TestCase
+{
+    public function dataProvider()
+    {
+        return [
+            ['01D85PP1982GF6KTVFHQ7W78FB', new Ulid('01d85pp1982gf6ktvfhq7w78fb')],
+        ];
+    }
+
+    /**
+     * @dataProvider dataProvider
+     */
+    public function testTransform($output, $input)
+    {
+        $transformer = new UlidToStringTransformer();
+
+        $input = new Ulid($input);
+
+        $this->assertEquals($output, $transformer->transform($input));
+    }
+
+    public function testTransformEmpty()
+    {
+        $transformer = new UlidToStringTransformer();
+
+        $this->assertSame('', $transformer->transform(null));
+    }
+
+    public function testTransformExpectsUlid()
+    {
+        $transformer = new UlidToStringTransformer();
+
+        $this->expectException(\Symfony\Component\Form\Exception\TransformationFailedException::class);
```

Comment:
please add a `use` statement for the class instead

Reply:
fixed